docs/real-world-examples/ccfraud.md

```diff
@@ -0,0 +1,50 @@
+# Finance example for Federated Learning in Azure ML
+
+This is a short example where we showcase possibilities of using Azure Machine Learning(AML) for training a model for credit card fraud detection in federating learning fashion. The example utilizes multitude of model architectures to demonstrate versatility of the proposed solution. We have chosen this scenario because we believe it is very common for credit card companies, banks and other similar financial institutions.
+
+:warning: This is an example for demonstration purposes only and the authors of this repository do not take any responsibility for loss or harm made by its usage.
```

Comment:
Do we want to keep this warning here? I don't have strong preference to keep this warning or not, but if we do decide to keep this warning, we might probably be consistent and keep same warning across all our 3 examples. It looks a bit weird to me that we used this warning in one example but not other 2 examples, since I see nothing special about this example? Or the warning is added simply because it is "financial" area? Thoughts @thomasp-ms @garg-amit @majercakdavid ?

Reply:
IMO we can remove this warning as we already have this note/warning `For the sake of simplicity, we will only provision an _open_ setup. Do not upload sensitive data to it!`